### DISCUSSION TITLE
Investigate using Stackblitz as a one size fits all replacement for Replit/CodeSandbox

### DISCUSSION THREAD

**Discussion by Asartea** [TheOdinProject/curriculum#26844] [Ideas]
Recently we have had several issues with the platforms we use for hosting example code: Replit added an on-by-default AI feature (#26551), and we've been having persistent problems with CodeSandbox showing the wrong file (#26719).

@damon314159  today linked to Stackblitz as an alternative. Looking at it quickly, it has some features that seem to solve a lot of these: there isn't any AI that I can find, it has an editor layout that looks very much like VSCode and therefore should be immediately recognizable, it has a clear config for setting which file to display by default (yah!), and lets you pull from not only a GitHub repo but also a specific path within that repo (yah!^2)

CC: @wise-king-sullyman

**Comment by wise-king-sullyman** [comment]
Hey @Asartea! Thanks for putting this discussion post together, and thanks @damon314159 for surfacing Stackblitz!

Just letting you know that I haven't had time to dig into this yet but that alternative is sounding quite good. Will dig into it once I have a chance, should be in the next week or two.

**Comment by damon314159** [comment]
I was reading vite docs and saw this "For a template at https://github.com/user/project, you can try it out online using https://github.stackblitz.com/user/project (adding .stackblitz after github to the URL of the project).". Could be good to quickly spin up demos when guiding learners
